### PR TITLE
[release-1.51] Backport ignore chown errors in additionalimagestore

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Backports parts of #1828 and also #1784 to address:
https://issues.redhat.com/browse/ACCELFIX-244

### CODE REVIEW

File: drivers/overlay/overlay.go
Comment (TomSweeneyRedHat) [inline drivers/overlay/overlay.go:1687]: @giuseppe please eyeball this.  In the picked commit, it uses `dir` instead of `workDirBase` as listed here in line 1682.  I think this is OK, but would like your verification.

**Comment by TomSweeneyRedHat** [comment]
Hold for now.  If tests pass, I'll add another commit to bump the version, then work on getting a release out.

**Comment by TomSweeneyRedHat** [comment]
@giuseppe PTAL, if I LGTY, I'll add a version bump so we can move forward with this. I'll also backport to the release-1.45 branch.

**Review by giuseppe — APPROVED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
Happy green test buttons here, and this is for a customer hot fix.  Can I get a merge please?

**Comment by TomSweeneyRedHat** [comment]
Given this has a couple of LGTM's and an /lgtm, I'm going to press the manual merge button.

**Pull request merged**
